Signal/src/views/OWSUnreadIndicatorCell.m

```diff
@@ -10,6 +10,8 @@
 #import "UIView+OWS.h"
 #import <JSQMessagesViewController/UIView+JSQMessages.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I took the opportunity to apply our code style to a few unrelated classes and fix a few build warnings.